src/test/java/teammates/test/cases/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -675,7 +717,7 @@ private void testDeleteInstructorAction() {
 
         ______TS("test the only registered instructor with the privilege to modify instructors cannot be deleted");
 
-        // Create an registered instructor with all privileges except modifying instructors
+        // Create a registered instructor with all privileges except modifying instructors
```

Comment:
No need to add this change to the PR; as this change is unrelated to the issue of visible instructors, adding this change may make tracing the history of this code via `git blame` (or other methods) more difficult.